### PR TITLE
Add hover effect to bio images

### PR DESCRIPTION
**Pull request by Ctannerweb**

### CODE REVIEW

**Comment by Ctannerweb** [comment]
@acacheung :eyes:

File: assets/stylesheets/modules/speakers.scss [hunk 1]
```diff
@@ -145,6 +149,37 @@
...
+  width: 100%;
+  height: 100%;
+  background-color: rgba($pink, .8);
+  border-radius: 50%;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  border: 0px solid $pink;
+  opacity: 0;
+  @include transition-duration(.25s);
+  @include transition-property(border,opacity);
```
Comment (acacheung) [inline assets/stylesheets/modules/speakers.scss]: one space after commas in parameters

File: assets/stylesheets/modules/speakers.scss [hunk 2]
```diff
@@ -145,6 +149,37 @@
...
+.speaker__bio__copy {
+  position: absolute;
+  width: 100%;
+  top: 50%;
+  font-size: 15px;
+  font-weight: $xbold;
+  color: $white;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+  @include translate(0,-50%);
```
Comment (acacheung) [inline assets/stylesheets/modules/speakers.scss]: one space after commas in parameters

File: assets/stylesheets/modules/speakers.scss [hunk 3]
```diff
@@ -145,6 +149,37 @@
...
+}
+
+.speaker__bio__info {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  background-color: rgba($pink, .8);
+  border-radius: 50%;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
```
Comment (acacheung) [inline assets/stylesheets/modules/speakers.scss]: mixin?

File: views/index.erb [hunk 4]
```diff
@@ -44,7 +44,12 @@
   <ul class="index__speakers">
     <li class="speaker--keynote">
       <a class="index__speaker__link" href="/speakers#chris-tse" title="Visit Chris Tse's Bio">
-        <img class="index__speaker__image" alt="Chris Tse" src="speakers/chris-tse.jpg">
+        <div class="l-speaker__bio">
+          <div class="speaker__bio__info">
+            <h3 class="speaker__bio__copy">read more</h3>
```
Comment (acacheung) [inline views/index.erb]: Read more

File: assets/stylesheets/modules/speakers.scss [hunk 5]
```diff
@@ -117,9 +121,9 @@
   max-width: 185px;
 }
 
-.index__speaker__image {
-  margin-bottom: 12px;
+.speaker__bio__image {
   box-sizing: border-box;
+  -webkit-box-sizing: border-box;
```
Comment (acacheung) [inline assets/stylesheets/modules/speakers.scss]: mixin

File: assets/stylesheets/modules/speakers.scss [hunk 6]
```diff
@@ -117,9 +121,9 @@
...
   border: 0px solid $pink;
```
Comment (acacheung) [inline assets/stylesheets/modules/speakers.scss]: no unit for zero

File: assets/stylesheets/modules/speakers.scss [hunk 7]
```diff
@@ -145,6 +149,37 @@
...
+
+.speaker__bio__info {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  background-color: rgba($pink, .8);
+  border-radius: 50%;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  border: 0px solid $pink;
```
Comment (acacheung) [inline assets/stylesheets/modules/speakers.scss]: no unit for zero

File: assets/stylesheets/modules/speakers.scss (hunk 1 above)
Comment (acacheung) [inline assets/stylesheets/modules/speakers.scss]: http://www.html5rocks.com/en/tutorials/speed/high-performance-animations/
scale is a higher performing animation. instead can we always have the border there and scale it in?

File: assets/stylesheets/modules/speakers.scss [hunk 8]
```diff
@@ -90,7 +90,11 @@
     .speaker__name {
       color: $pink;
     }
-    .index__speaker__image {
+    .speaker__bio__image {
+      border: 6px solid $pink;
+    }
+    .speaker__bio__info {
+      opacity: 1;
       border: 6px solid $pink;
```
Comment (acacheung) [inline assets/stylesheets/modules/speakers.scss]: do we need this line anymore?

File: assets/stylesheets/modules/speakers.scss [hunk 9]
```diff
@@ -117,10 +121,9 @@
...
 
-.index__speaker__image {
-  margin-bottom: 12px;
-  box-sizing: border-box;
-  border: 0px solid $pink;
+.speaker__bio__image {
+  border: 0 solid $pink;
+  @include box-sizing(border-box);
   @include transition-duration(.25s);
   @include transition-property(border);
```
Comment (acacheung) [inline assets/stylesheets/modules/speakers.scss]: only border-width i think

File: assets/stylesheets/modules/speakers.scss (hunk 8 above)
Comment (Ctannerweb) [inline assets/stylesheets/modules/speakers.scss]: With a pink border and ping background nope we don't need it. The opacity wont mess with the border color now.

File: assets/stylesheets/modules/speakers.scss [hunk 10]
```diff
@@ -90,9 +90,12 @@
     .speaker__name {
       color: $pink;
     }
-    .index__speaker__image {
+    .speaker__bio__image {
       border: 6px solid $pink;
```
Comment (acacheung) [inline assets/stylesheets/modules/speakers.scss]: means this can be just border-width: 6px

**Comment by Ctannerweb** [comment]
@acacheung :eyes:

**Pull request merged**